### ISSUE TITLE
Multiple vulnerabilities in Ruby on Rails - CVE-2021-22880 CVE-2021-22881

### ISSUE THREAD

**Issue by aguerson** [bigbluebutton/greenlight#2526]
```
    https://groups.google.com/g/rubyonrails-security/c/zN_3qA26l6E


    https://groups.google.com/g/rubyonrails-security/c/ZzUqCh9vyhI

    CVE CVE-2021-22880
    http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22880

    CVE CVE-2021-22881
    http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-22881
```

**Comment by defnull** [comment]
Do you have any details on how these might affect Greenlight? Both CVEs seem to be irrelevant here.

**Comment by aguerson** [comment]
What ?

Greenlight use ROR, it is enough for me.
I just advice you to update it.
You can't let a vulnerable composant. It is not acceptable !

**Comment by ichdasich** [comment]
So, to give some contex:  
[CVE-2021-22880] does not affect greenlight. This is specific to money-type'd input fields, which GL does not have.  
[CVE-2021-22881] _might_ affect greenlight, if SAFE_HOSTS in .env feeds into this and users specified the safe host(s) with a leading dot. (If i got the CVE correctly)

Indeed, i'd argue that an update due to [CVE-2021-22881] might be a thing to do here.

**Comment by defnull** [comment]
> You can't let a vulnerable composant. It is not acceptable!

Yes, dependencies should be kept up to date. But there is a difference:
- A vulnerability that can be exploited right now requires an immediate investigation and a security patch release ASAP. Please follow responsible disclosure best practices and NOT post these in a public issue tracker. You are putting everyone at risk by disclosing such issues without giving the project owners some time to prepare a fix.
- A bug in some unused part of RoR that does not affect Greenlight at all and cannot be exploited, however, would be okay to fix with a regular release after proper testing. No need to rush or panic.

**Comment by ichdasich** [comment]
Well, let's keep some healthy threat modelling in mind, and cool down a little. The current RoR thinggy (even the one possibly affecting GL) does not really warrant a responsible disclosure process.

I think the more fundamental question here is how to do supplychain vulnerability management for GL. And there might indeed be some room for improvement (also see the Alpine ticket); Buuut... that is another issue. Until then, it might be good to bump a version number in a gemfile and close this (and maybe get a ticket out to take a deeper look at how dependencies might be better managed... )

**Comment by farhatahmad** [comment]
> Well, let's keep some healthy threat modelling in mind, and cool down a little.

Agreed :). Both sides are correct here - Having dependencies with vulnerabilities is not ideal, but these are not "critical" issues as neither affects Greenlight. The way we tend to approach these scenarios is fix it in the next "regular" release (which we normally do on Wednesdays). If this was critical, we would release a new version right away

**Comment by farhatahmad** [comment]
Fixed in #2553
